service/src/main/java/com/commercetools/pspadapter/payone/util/PayoneHash.java

```diff
@@ -17,6 +18,6 @@
      */
     @SuppressWarnings("deprecation")
     public static String calculate(String in) {
-        return Hashing.md5().hashString(in, Charsets.UTF_8).toString();
+        return DigestUtils.md5Hex(in);
```

Comment:
This needs to be tested I guess.

Reply:
There's a test already: https://github.com/commercetools/commercetools-payone-integration/blob/ed35e69adeb51a09e5fdd099c0ce2e8ceadb73b7/service/src/test/java/com/commercetools/pspadapter/payone/config/PayoneConfigTest.java#L73